components/footer/footermenu.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from 'react'
```

Comment:
rename this file to footer-menu.tsx
Also there is no reason to have it in footer directory since this is the only file there. just have it in components.

Reply:
done
footer folder removed
footer-menu.tsx moved to compomemts